.travis.yml

```diff
@@ -11,6 +14,15 @@ node_js:
 services:
   - redis-server
 
+before_install:
+  # Use a specific version of yarn in CI. This ensures yarn.lock format doesn't change.
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.19.1
+  - export PATH="$HOME/.yarn/bin:$PATH"
+
+install:
+  # ensure unexpected changes to yarn.lock break the build
+  - yarn install --frozen-lockfile --non-interactive
```

Comment:
`--frozen-lockfile` is generally recognized as the preferred way to install dependencies in a CI environment.

https://yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install-frozen-lockfile
https://github.com/yarnpkg/yarn/issues/4147
https://github.com/cypress-io/circleci-orb/issues/72